src/types/Document.ts

```diff
@@ -27,6 +27,152 @@ export type Document = {
+  archive_serial_number__isnull: boolean;
+  /** Created year of document is */
+  created__year: number;
+  /** Created month of document is */
+  created__month: number;
+  /** Created day of month of document is */
+  created__day: number;
+  /** Created date of document is greater than */
+  created__date__gt: string;
+  /** Created date of document is greater than */
+  created__date__lt: string;
+  /** Created is grater than */
+  created__gt: string;
+  /** Created is lower than */
+  created__lt: number;
+  /** Added year is */
+  added__year: number;
+  /** Added month is */
+  added__month: number;
+  /** Added day of month is */
+  added__day: number;
+  /** Added date is greater than */
+  added__date__gt: string;
+  /** Added date is lower than */
+  added__date__lt: string;
+  /** Added is greater than */
+  added__gt: string;
+  /** Added is lower than */
+  added__lt: string;
+  /** Modified year is */
+  modified__year: number;
+  /** Modified month is */
+  modified__month: number;
+  /** Modified day of month */
+  modified__day: number;
+  /** Modified date is greater than */
+  modified__date__gt: string;
+  /** Modified date is lower than */
+  modified__date__lt: string;
+  /** Modified is greater than */
+  modified__gt: string;
+  /** Modified is lower than */
+  modified__lt: string;
+  /** Correspondent is null */
+  correspondent__isnull: boolean;
+  /** Correspondent id is */
+  correspondent__id: CorrespondentId;
+  /** TODO: comma sperated list? arry or string?  Correspondent id is in */
+  correspondent__id__in: number[];
+  /** Correspondent name starts with */
+  correspondent__name__istartswith: string;
+  /** Correspondent name ends with */
+  correspondent__name__iendswith: string;
+  /** Correspondent name contains */
+  correspondent__name__icontains: string;
+  /** Correspondent name is exactly */
+  correspondent__name__iexact: string;
+  /** Tags id is */
+  tags__id: number;
+  /** TODO: comma sperated list? arry or string? Tags id is in */
+  tags__id__in: number[];
+  /** Tags name starts with */
+  tags__name__istartswith: string;
+  /** Tags name ends with */
+  tags__name__iendswith: string;
+  /** Tags name contains */
+  tags__name__icontains: string;
+  /** Tags name is exactly */
+  tags__name__iexact: string;
+  /** Document type is null */
+  document_type__isnull: boolean;
+  /** Document type id is */
+  document_type__id: DocumentTypeId;
+  /**  TODO: comma sperated list? arry or string? Document type id is in */
+  document_type__id__in: DocumentTypeId[];
+  /** Document type name start width */
+  document_type__name__istartswith: string;
+  /** Document type name ends width */
+  document_type__name__iendswith: string;
+  /** Document type name contains */
+  document_type__name__icontains: string;
+  /** Document type name is exactly */
+  document_type__name__iexact: string;
+  /** Document is tagged */
+  is_tagged: boolean;
+  /** TODO:  */
+  tags__id__all: number;
+  /** TODO:  */
```

Comment:
What needs to be done here?

Reply:
There is still a description to be added in the comment, unfortunately I don't know exactly what the filter does.